lib/smb.php

```diff
@@ -39,30 +39,28 @@ public function __construct($host) {
 	private function tryAuthentication($uid, $password) {
 		$uidEscaped = escapeshellarg($uid);
 		$password = escapeshellarg($password);
-		$command = self::SMBCLIENT.' '.escapeshellarg('//' . $this->host . '/dummy').' -U'.$uidEscaped.'%'.$password;
+		$command = self::SMBCLIENT.' '.escapeshellarg('//' . $this->host . '/dummy').' -U '.$uidEscaped.'%'.$password;
```

Comment:
https://explainshell.com/explain/1/smbclient make sense :+1:

Reply:
where can you see her that the space is needed?
maybe the safer option would be to use `--user=username`?

Reply:
> where can you see her that the space is needed?

Added a link with proper information :see_no_evil:

Reply:
okay thank you very much, seems legit :)